website/source/docs/guides/containers-guide.md

```diff
@@ -0,0 +1,328 @@
+server-1  172.17.0.2:8301  alive   server  1.4.3  2         dc1  <all>
+client-1  172.17.0.3:8301  alive   client  1.4.3  2         dc1  <default>
+
+```
+
+Now that you have a small cluster, you can register a service and 
+perform maintenance operations. 
+
+## Register a Service
+
+Start a service in a third container and register it with the Consul client. The basic service increments a number every time it is accessed and returns that number. 
+
+Pull the container and run it with port forwarding so that you can access it from your web browser by visiting [http://localhost:9001](http://localhost:9001).
+
+```sh
+$ docker pull hashicorp/counting-service:0.0.2
+$ docker run \
+   -p 9001:9001 \
+   -d \
+   --name=weasel \
+   hashicorp/counting-service:0.0.2
+```
+
+Next, you will register the counting service with the Consul client by adding a service definition file called `counting.json` in the directory `consul/config`.
+
+```sh
+$ docker exec fox /bin/sh -c "echo '{\"service\": {\"name\": \"counting\", \"tags\": [\"go\"], \"port\": 9001}}' >> /consul/config/counting.json"
+```
+
+Since the Consul client does not automatically detect changes in the 
+configuration directory, you will need to issue a reload command for the same container.
+
+```sh
+$ docker exec fox consul reload
+Configuration reload triggered
+```
+
+If you go back to the terminal window where you started the client, you should see logs showing that the Consul received the hangup signal, is reloading its configuration, and has synced the counting service.
+
+```sh
+2019/07/01 21:49:49 [INFO] agent: Caught signal:  hangup
+2019/07/01 21:49:49 [INFO] agent: Reloading configuration...
+2019/07/01 21:49:49 [INFO] agent: Synced service "counting"
+```
+
+### Use Consul DNS to Discover the Service
+
+Now you can query Consul for the location of your service using the following dig command against Consul's DNS.
+
+```sh
+$ dig @127.0.0.1 -p 8600 counting.service.consul
+
+; <<>> DiG 9.10.6 <<>> @127.0.0.1 -p 8600 counting.service.consul
+; (1 server found)
+;; global options: +cmd
+;; Got answer:
+;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 47570
+;; flags: qr aa rd; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 2
+;; WARNING: recursion requested but not available
+
+;; OPT PSEUDOSECTION:
+; EDNS: version: 0, flags:; udp: 4096
+;; QUESTION SECTION:
+;counting.service.consul.       IN      A
+
+;; ANSWER SECTION:
+counting.service.consul. 0      IN      A       172.17.0.3
+
+;; ADDITIONAL SECTION:
+counting.service.consul. 0      IN      TXT     "consul-network-segment="
+
+;; Query time: 1 msec
+;; SERVER: 127.0.0.1#8600(127.0.0.1)
+;; WHEN: Tue Jul 02 09:02:38 PDT 2019
+;; MSG SIZE  rcvd: 104
+```
+
+You can also see your newly registered service in Consul's UI, [http://localhost:8500](http://localhost:8500).
+
+![Consul UI with Registered Service](/assets/images/consul-containers-ui-services.png 'Consul UI with Registered Service')
```

Comment:
Does this image need to be included with the PR?

Reply:
Not right now, it's in the Learn repo.